src/shared.js

```diff
@@ -1988,7 +1987,7 @@ export const locationConceptIDToLocationMap = {
   [conceptIds.nameToKeyObj.hpPN]: {
     siteSpecificLocation: 'HP Park Nicollet',
     siteAcronym: 'HP',
-    siteCode: nameToKeyObj.hPartners,
+    siteCode: `${healthProviderAbbrToConceptIdObj.healthPartners}`,
```

Comment:
Do we need string here?

Reply:
The other siteCode values use concept Ids with string values. I had to change the date type from number to string.